### DISCUSSION TITLE
Is there a way to stop PMM from running automaticly each night? (Unraid Docker)

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#1092] [Q&A] [answered]
Hello,
I have PMM in an unraid docker and start the container by hand.

Once, the container is started, I only let PMM run when I want to test my configs or want PMM to create new collections.
To start PMM, I do something like `
docker exec -it pmm python plex_meta_manager.py --run -c config/config_mm.yml --no-missing` on the command line of an unraid shell.

Now I realized, that the docker runs every night at 05:00 too, but I don't want PMM to run automaticly.

I want it run PMM only, when I start it from the command line (as above) and the docker container is running too.

I checked [here](https://metamanager.wiki/en/latest/home/environmental.html#ignore-schedules) to ignore schedules, but those seem to be the scheduled collections only, which I want to run, IF I start the container AND run PMM from the command line.

I also checked other "Run Commands & Environment Variables" but have not found something suitable for me.

I removed the "Time to run" from the docker container configuraion, shown in the [unraid walkthrough](https://metamanager.wiki/en/latest/home/guides/unraid.html), but that did not seem to change anything.

The problem is, when I run PMM by hand at 22:00 and it takes, say, 8 hours to complete, the automatic run at 05:00 overwrites the log files with errors.
That, it seems, is because I had not configured a `config.yml`, only my personal config `config_mm.yml` (as above).
So I added a `config.yml` with a minimal valid config, but that throws errors too, when another PMM instance (my manually started) is running.

So, my question is, how can I stop PMM to run automaticly and only, when I start it by hand via the above command?

Is this possible?

Thank you very much.

**EDIT**: I know, the recommended way is to run PMM via a scheduled run each night, so everything set up runs smoothly and automaticly. But my configs are far away from being finished and so I don't want/need PMM to run automaticly yet.

**Comment by meisnate12** [comment, marked as answer]
pmm never overwrites a log when you "run PMM by hand at 22:00" youre actually just setting it up to run when scheduled. You need to use the `--run` command from the `Run Commands & Environment Variables` Page

**Reply by gitttt54** [reply]
>pmm never overwrites a log when you "run PMM by hand at 22:00"

Yes, thank you, that is not what is overwritting the log.

The log gets overwritten by the PMM process, which starts _automaticly_ at 05:00.
I can see it very clearly, because of the logfiles timestamp and the timestamp inside the log.

I use the `--run` when I manually start PMM as stated in my first post.

I don't want PMM to start automaticly at 05:00 (or any other time). Is this possible to set up?

Also, I have no _Extra Parameters_ or _Post Arguments_ in my container configuration, maybe I should put something in there to NOT have PMM start automaticly?

This is my container config:
<img width="1462" alt="cfg1" src="https://user-images.githubusercontent.com/96403800/194376280-735cf14d-0c5d-4e58-a7cd-68618b4fb568.png">
<img width="1463" alt="cfg2" src="https://user-images.githubusercontent.com/96403800/194376295-6597e871-6521-4b89-b3f9-2e5702df0fb6.png">

**Reply by chazlarson** [reply]
If you don't want it to run automatically, don't have a container running constantly.

If the container is started [without the PMM_RUN flag], then PMM is waiting to wake up at the times you specify.  You can [set the time that happens](https://metamanager.wiki/en/latest/home/environmental.html#time-to-run), but you can't turn that behavior off.

[EDIT: I suppose you could schedule everything to happen at times *other than* 5 AM, so it would fire up at 5AM and find nothing to do]

If you create your container WITH the [run flag](https://metamanager.wiki/en/latest/home/environmental.html#run) enabled, when the container starts it will run immediately and then exit.  Probably this is what you want.  Create such a container [or modify your existing one] and then you can start the container when you want to run PMM.  It will then run, exit, and won't then be running all the time and waiting to wake up.

When your config is set and you want to let it fire up daily, remove that flag to reenable the daily run behavior.

Also, the logs are rotated, so when it fires up automatically, it's creating a new meta.log and rotating the rest so the last ten are always available.
